File: mysql-testing-files/Test-Query.js
let mysql = require('mysql');

let con = mysql.createConnection({
  host: "localhost",
  user: "PieniYritys",
  password: "PieniYritys123,",
  database: "Pieniyritys"
});

con.connect(function(err) {
  if (err) throw err;
  con.query("SELECT * FROM Tyontekija", function (err, result) {
    if (err) throw err;
    console.log(result);
  });
  console.log("Connected");
});

File: mysql-testing-files/Test-Query.py
import mysql.connector

mydb = mysql.connector.connect(
  host="localhost",
  user="PieniYritys",
  password="PieniYritys123,",
  database="PieniYritys"
)

mycursor = mydb.cursor()

mycursor.execute("SELECT * FROM Tyontekija")

myresult = mycursor.fetchall()

for x in myresult:
  print(x)